doc/sphinx-guides/source/admin/external-tools.rst

```diff
@@ -115,7 +115,7 @@ Dataset level explore tools allow the user to explore all the files in a dataset
 Dataset Level Configure Tools
 +++++++++++++++++++++++++++++
 
-Configure tools at the dataset level are not currently supported.
+Dataset level configure tools can be launched by users who have edit access to the dataset and are found under the "Edit Dataset" menu.
```

Comment:
maybe two sentences here. a quick read might make you think that the users are found under the the Edit Dataset menu.

Reply:
Ha, yes. I agree. Two sentences now: f9fba20